content/docs/setup/kubernetes/quick-start/index.md

```diff
@@ -28,6 +28,12 @@ To install and configure Istio in a Kubernetes cluster, follow these instruction
 
 1. Check the [Requirements for Pods and Services](/docs/setup/kubernetes/spec-requirements/).
 
+## Production install
```

Comment:
Please do not add this here. we are already providing additional guidance in a new landing page:
https://docs.google.com/document/d/1b6j7F8ogtC1DAJIK9fSaPPCSAvKaUIgh8t8g2wnln_c/edit?ts=5c758b15
The content of this section conflicts with that effort.